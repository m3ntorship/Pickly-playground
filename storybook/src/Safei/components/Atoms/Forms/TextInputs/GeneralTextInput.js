import React from "react";

const GeneralTextInput = () => {
  return (
    <div className="inline-block">
      <input
        placeholder="What do you want to ask about?"
        className="px-xl py-l rounded-md border  border-grey-shd5 placeholder-grey-shd1 h-8  w-96 hover:border-grey-shd1  outline-none focus:outline-none focus:border-dark"
      />
    </div>
  );
};

export default GeneralTextInput;
