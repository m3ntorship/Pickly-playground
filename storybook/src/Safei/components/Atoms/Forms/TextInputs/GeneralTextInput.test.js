import React from "react";
import renderer from "react-test-renderer";
import GeneralTextInput from "./GeneralTextInput";

describe("GeneralTextInput field", () => {
  it("renders a marked input ", () => {
    const tree = renderer.create(<GeneralTextInput />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
