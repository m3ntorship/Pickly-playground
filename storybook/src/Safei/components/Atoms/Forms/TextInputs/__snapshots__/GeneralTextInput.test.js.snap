// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GeneralTextInput field renders a marked input  1`] = `
<div
  className="inline-block w-full"
>
  <input
    className="px-xl py-3 rounded-md border  border-grey-shd5 placeholder-grey-shd1 w-full hover:border-grey-shd1  outline-none focus:outline-none h-11 focus:border-dark"
    placeholder="What do you want to ask about?"
  />
</div>
`;
